sway-types/src/source_engine.rs

```diff
@@ -123,4 +123,9 @@ impl SourceEngine {
         v.sort();
         v
     }
+
+    pub fn get_source_id_from_module_id(&self, module_id: ModuleId) -> Option<BTreeSet<SourceId>> {
```

Comment:
`BTreeSet` confused me at first, because I've expected a single `SourceId` based on the method name. Renaming it to plural will remove the confusion and precisely convey that a module can consist of a more then one file.
```suggestion
    pub fn get_source_ids_from_module_id(&self, module_id: ModuleId) -> Option<BTreeSet<SourceId>> {
```